addon/-private/helpers/-format-base.js

```diff
@@ -31,7 +30,7 @@ const AbstractHelper = Helper.extend({
 
   compute([value], options) {
     if (isEmpty(value)) {
-      if (getWithDefault(options, 'allowEmpty', this.allowEmpty)) {
+      if (options.allowEmpty || this.allowEmpty) {
```

Comment:
```suggestion
      if (options.allowEmpty ?? this.allowEmpty) {
```

Otherwise `options.allowEmpty = false` would not work.